packages/asset/test/fixtures/catalystFixture.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  deployments,
+  getUnnamedAccounts
+} from "hardhat";
+
+export const runCatalystSetup = deployments.createFixture(
+  async ({ deployments, getNamedAccounts, ethers }) => {
+    await deployments.fixture(["Catalyst"]);
```

Comment:
Please use new approach where we don't rely on deployments / deploy scripts (i.e. we deploy a TestCatalyst contract in this fixture) like in this [example](https://github.com/thesandboxgame/sandbox-smart-contracts/blob/80df7d7be995c533e675b1dc07d0f2b80b6ab84e/packages/example_hardhat/test/fixtures.ts#L15)